src/seedu/addressbook/data/person/Birthday.java

```diff
@@ -0,0 +1,64 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's date of birth in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidBirthday(String)}
+ */
+
+public class Birthday {
+
+
+    public static final String EXAMPLE = "01/09";
+    public static final String MESSAGE_BIRTHDAY_CONSTRAINTS =
+            "Person date of birth should be two numerals separated by '/'";
+    public static final String BIRTHDAY_VALIDATION_REGEX = "[\\d{2}]+/[\\d{2}]+";
```

Comment:
For the checking of birthday, you can just put it to the Date and see whether it has any errors. As far as I know, this might not be a correct regex for checking dates :)